recipes/autoconf/all/patches/2.71-0002-no-perl-path-in-shebang.patch

```diff
@@ -1,48 +0,0 @@
---- bin/autoheader.in
```

Comment:
autoconf can replace `@PERL@` with a system dependent perl path.
This results in the autoconf recipe not being portable anymore.

e.g. on Windows, the perl path will be a path to the perl interpreter.
That same recipe will not work on Linux anymore.

Would changing the shebang to `/usr/bin/env perl` be fine as well?

Reply:
Yes, `/usr/bin/env perl` is also fine (at least on the systems I use). I wasn't aware that the resulting recipe might end up on different hosts.

Reply:
Perhaps, it's not a supported usecase by the autotools developers.
But since they are script-only, why not try it.
Apart from these patches, no big changes are required.

Reply:
Updated the patch.